supplementary_materials/notebooks/phi3_rag_on_client.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # ! pip install optimum[openvino,nncf]
</pre><p>maybe better to not  provide this line for avoiding user confusion. We can update command, when optimum-intel will be released with phi-3 support </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2025/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>